packages/atlas/src/views/viewer/VideoView/VideoView.styles.ts

```diff
@@ -116,17 +116,23 @@ export const DetailsWrapper = styled.div`
 `
 
 export const DescriptionContainer = styled.div`
-  margin-top: ${sizes(6)};
-  margin-bottom: ${sizes(8)};
-
-  ${media.md} {
-    margin: ${sizes(8)} 0;
-  }
+  margin: ${sizes(6)} 0;
 `
 export const DescriptionTitle = styled(Text)`
   margin-bottom: ${sizes(2)};
 `
 
+export const DescriptionBody = styled.div<{ detailsExpanded?: boolean }>`
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  /* stylelint-disable-next-line value-no-vendor-prefix */
+  display: -webkit-box;
+  -webkit-line-clamp: ${({ detailsExpanded }) => (detailsExpanded ? 'unset' : 3)};
```

Comment:
Is this prefix necessary?

Reply:
Unfortunately yes, if I remove this, the prefix won't be added by emotion. This is what I'll see in dev tools
![image](https://user-images.githubusercontent.com/51168865/163979989-717c8ff1-aa24-44f6-865f-12cd5e86e714.png)